packages/erxes-ui/src/components/richTextEditor/extensions/Table/Row/Row.ts

```diff
@@ -7,6 +7,9 @@ export const Row = TiptapRow.extend({
       style: {
         parseHTML: (element) => element.getAttribute('style'),
       },
+      align: {
+        parseHTML: (element) => element.getAttribute('align'),
+      },
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **question (code_clarification):** Check the Row extension for proper handling of alignment attributes.